wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -617,6 +617,7 @@ test-suite wallet-new-specs
                     , formatting
                     , hedgehog
                     , hspec
+                    , insert-ordered-containers
```

Comment:
We don't need this import right?

Reply:
I always try to use `Control.Lens.At` instead of bringing in this dep :/ ...

Reply:
Or `Control.Lens.Indexed` in that case actually

Reply:
I needed to get access to the keys of the map and couldn't figure out how to do that with lens :\